test/interp/return-call-indirect.txt

```diff
@@ -0,0 +1,58 @@
+;;; TOOL: run-interp
+(module
+  (type $iii_i (func (param i32 i32 i32)(result i32)))
+  (table anyfunc (elem $facInd))
+
+  (func (export "call") (result i32)
```

Comment:
`call` and `helper` seem unrelated to this test?

Reply:
removed